app/controllers/unavailable_on_route_controller.rb

```diff
@@ -0,0 +1,43 @@
+###
+# Copyright 2016 - 2019 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/master/LICENSE.md
+###
+
+class UnavailableOnRouteController < ApplicationController
+  before_action :authenticate_user!
+  before_action :some_clients_editable!
+  before_action :set_client
+  before_action :set_unavailable_for
+
+  def destroy
+    if @unavailable_for.destroy
+      flash[:notice] = "#{@client.name} is now available to match on the #{@unavailable_for.route&.title}"
+    else
+      flash[:error] = "Unable to make #{@client.name} available on the #{@unavailable_for.route&.title}"
+      render :new
```

Comment:
Where will this end up?

Reply:
No where!   Good catch, that was a bad copy paste job.